utils/logger.py

```
@@ -2,33 +2,37 @@
 import os
 import datetime
 
+
 def log(message):
     """Log message to both file and console with timestamp"""
-    timestamp = datetime.datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S UTC')
+    timestamp = datetime.datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S UTC")
```

Ruff (0.11.9)

`datetime.datetime.utcnow()` used

(DTZ003)